op-batcher/batcher/channel_builder.go

```diff
@@ -83,15 +83,15 @@ func (cc *ChannelConfig) Check() error {
 	// will infinitely loop when trying to create frames in the
 	// [channelBuilder.OutputFrames] function.
 	if cc.MaxFrameSize == 0 {
-		return ErrZeroMaxFrameSize
+		return errors.New("max frame size cannot be zero")
```

Comment:
any reason to get rid of the sentinel error?

Reply:
probably because of a comment I made in another pr at https://github.com/ethereum-optimism/optimism/pull/5167#discussion_r1141833350